assets/stylesheets/tint.scss

```diff
@@ -1,135 +1,177 @@
-$input-width: 20rem;
+@import "variables";
+@import "mixins";
+@import "base";
 
-* {
-	box-sizing: border-box;
-}
-
-%card {
-	background: #f0f0f0;
-	border: 1px solid #ccc;
-	border-radius: 2px;
-	display: block;
-	margin-bottom: 2rem;
-	max-width: 100%;
-	overflow: auto;
-	padding: 1rem;
-
-	:last-child {
-		margin-bottom: 0;
-	}
-}
-
-.card {
-	@extend %card;
+a {
+	color: $link-color;
 }
 
 body {
-	font-family: sans-serif;
-	padding: 2rem;
+	display: flex;
+	font-family: $font-stack;
+	font-size: 16px;
```

Comment:
Do we have to execute a reset on base font size, instead of allowing the user's local preferences to rule the day?